test_rmw_implementation/test/test_subscription.cpp

```diff
@@ -150,6 +150,14 @@ TEST_F(CLASSNAME(TestSubscription, RMW_IMPLEMENTATION), create_with_bad_argument
   EXPECT_EQ(nullptr, sub);
   rmw_reset_error();
 
+  rosidl_message_type_support_t * non_implementation_ts = const_cast<rosidl_message_type_support_t *>(ts);
+  const char * old_identifier = non_implementation_ts->typesupport_identifier;
```

Comment:
@Lobotuerk nit: 
```suggestion
  const char * typesupport_identifier = non_implementation_ts->typesupport_identifier;
```
Same elsewhere.

Reply:
Updated on f1fc248

Reply:
Updated on 345697b